packages/web/src/components/result/addons/Pagination.js

```diff
@@ -33,8 +33,8 @@ const buildPaginationDOM = (props, position) => {
 					? start + (Math.ceil(pages / 2) - (pages % 2))
 					: totalPages + 1;
 		}
-		if (currentPage > (totalPages - pages) + 2) {
-			start = (totalPages - pages) + 2;
+		if (currentPage > totalPages - pages + 2) {
+			start = totalPages - pages + 2;
```

Comment:
@jyash97 Is your eslint with prettier off, this line has the linting error?
Why there are a lot of unnecessary changes?
Can you update it?

Reply:
There are eslint errors on this file and were there from before I didn't fix them because it would have created more noise in the PR we can do a follow-up PR to fix these errors. They are mainly because of `no-mix-operators`